### PR TITLE
Add additional keywords for Elixir

### PR DESCRIPTION
**Pull request by ben-dyer**
Add `defmacro`, `quote` and `unquote` keywords for elixir (cf. https://hexdocs.pm/elixir/1.12/syntax-reference.html#the-elixir-ast) and also added `defn` for numerical definition (https://github.com/elixir-nx/nx/tree/main/nx#numerical-definitions)

### CODE REVIEW

**Review by RunDevelopment — CHANGES_REQUESTED** [review]
Thank you for the PR @ben-dyer!

Please add the new keywords to Elixir's [keyword test](https://github.com/PrismJS/prism/blob/master/tests/languages/elixir/keyword_feature.test) (more [info](https://prismjs.com/test-suite.html#writing-tests-insert-and-update)).

Also, we commit the minified files as well. Please run `npm ci && npm run build` and commit the changes.

**Comment by ben-dyer** [comment]
@RunDevelopment Thanks for the help :bow:

File: tests/languages/elixir/keyword_feature.test [hunk 1]
```diff
@@ -6,6 +6,8 @@ defimpl defmodule
 defp
 defprotocol
 defdelegate
+defmacro quote unquote
+defn defnp
```
Comment (RunDevelopment) [inline tests/languages/elixir/keyword_feature.test:10]: The expected JSON (the one below) also has to be updated. Just run `npm run test:languages -- --update`.

Comment (ben-dyer) [reply tests/languages/elixir/keyword_feature.test]: Thank you - should be fixed in 275542d

**Comment by RunDevelopment** [comment]
Nice.

Thank you for contributing @ben-dyer!

**Pull request merged**